dl1_data_handler/reader.py

```diff
@@ -961,6 +961,53 @@ def _transform_to_sky_spher_offsets(self, table) -> Table:
         table.add_column(angular_separation, name="angular_separation")
         return table
 
+    def get_parameters_dict(self, batch, parameter_list) -> Dict:
```

Comment:
As discussed, please just change the returning of `get_parameters()` if returning a `dict` is more convenient for you.